pkgs/development/python-modules/versiontag/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchFromGitHub, setuptools, git }:
+
+buildPythonPackage rec {
+  pname = "versiontag";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "thelabnyc";
+    repo = "python-versiontag";
+    rev = "r${version}";
+    sha256 = "1axv2214ykgv5adajv10v2zy5fr9v77db54rkik6ja29p66zl90n";
+  };
+
+  checkInputs = [ git ];
+  buildInputs = [ setuptools ];
```

Comment:
Ups, missed that one. `setuptools` are usually not needed to be part of `buildInputs`.

Reply:
You're right, it builds and run without setuptools.
Changes have been done.

Reply:
You're right, it builds and runs without it.
Changes have been made.